session.go

```diff
@@ -43,3 +44,12 @@ func (s *Session) FlushStorage() (err error) {
 	_, err = synchronousEvent(s.ctx, s, s.w, Event{Name: EventNameSessionCmdFlushStorage, TargetID: s.id}, EventNameSessionEventFlushedStorage)
 	return
 }
+
+// Loads a chrome extension
+func (s *Session) LoadExtension() (err error) {
+	if err = s.ctx.Err(); err != nil {
+		return
+	}
+	_, err = synchronousEvent(s.ctx, s, s.w, Event{Name: EventNameSessionCmdLoadExtension, TargetID: s.id})
```

Comment:
You need to add `, EventNameSessionEventLoadedExtension` at the end of this call, since as `loadExtension` call is asynchronuous, we need to wait for it to be over.